### ISSUE TITLE
API section: Accessibility: [MAS1.4.3]:Text elements must have sufficient color contrast against the background.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3821]
**Build details:**

**OS Version:** 2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)

**URL:** <https://fluidframework.com/apis/>

**Pre-Requisite:**

Add Accessibility Insights extension for web to Edge Anaheim (url:[https://accessibilityinsights.io/](https://accessibilityinsights.io/))

  

**Repro Steps:**

1\. Open the given URL with valid credentials.

2.Now, Run fastpass using Accessibility Insights for Web and observe the failure instances of color-contrast

  
**Actual Result**  
Text elements are not having sufficient color contrast ratio.  

  
**Expected Result:**  
Text elements must have sufficient color contrast against the background.  

  

**Note:** This issues is observed for all the "List of.." texts above the all the tables in the page.

  

**User Impact:**  

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.  

  

**MAS Reference:**

**MAS 1.4.3 - Contrast (Minimum)**

**Comment by micahgodbolt** [comment]
hah...off by 3/100th 

![image](https://user-images.githubusercontent.com/1434956/95787380-46581880-0c8e-11eb-8db5-2292b6123b44.png)
